codeHF/config_tasks.sh

```diff
@@ -45,6 +45,7 @@ DOO2_CAND_2PRONG=0  # hf-candidate-creator-2prong
 DOO2_CAND_3PRONG=0  # hf-candidate-creator-3prong
 DOO2_CAND_CASC=0    # hf-candidate-creator-cascade
 DOO2_CAND_X=0       # hf-candidate-creator-x
+DOO2_CAND_LB=0      # hf-candidate-creator-lb
```

Comment:
```suggestion
DOO2_CAND_LB=0      # hf-candidate-creator-lb
DOO2_CAND_X=0       # hf-candidate-creator-x
```
Lb is before X everywhere else.